src/vega.schema.ts

```diff
@@ -17,13 +17,15 @@ export type VgParentRef = {
 
 export type VgFieldRef = string | VgParentRef | VgParentRef[];
 
+export type VgOpSort = boolean | {
```

Comment:
VgSortField? 

The intention for this is to sort by another field, not by the scale's field itself.  
Then you need to add Op to disambiguate which value of the other field you want to use. 

Thus, the matter here is that it's a sort field.  Op is secondary.